src/main/java/org/apache/xml/security/stax/impl/processor/output/AbstractSignatureOutputProcessor.java

```diff
@@ -181,8 +182,8 @@ protected void setActiveInternalSignatureOutputProcessor(
     protected DigestOutputStream createMessageDigestOutputStream(String digestAlgorithm)
             throws XMLSecurityException {
 
-        String jceName = JCEAlgorithmMapper.translateURItoJCEID(digestAlgorithm);
-        String jceProvider = JCEAlgorithmMapper.getJCEProviderFromURI(digestAlgorithm);
+        String jceName = JCEMapper.translateURItoJCEID(digestAlgorithm);
+        String jceProvider = JCEMapper.getJCEProviderFromURI(digestAlgorithm);
```

Comment:
Why this change?

Reply:
Automatic change made by Eclipse - the method is defined in the parent class, but it is static. There is no such method in `JCEAlgorithMapper`. In fact the `JCEAlgorithmMapper` should NOT inherit from `JCEMapper`, both contain just static methods and both should be final. But that would be a breaking change, however I don't think anyone "extended" these classes.

I would keep it there as it is here, because it is the right direction. In "sanct4" branch we can continue with breaking changes.